spec/takeaway_spec.rb

```diff
@@ -0,0 +1,61 @@
+require 'takeaway'
+
+describe Takeaway do
+
+  let(:menu){ Menu.new({"soup"=>3.22, "garlic bread"=>1.35, "olives"=>2.86})}
+  subject(:takeaway) {described_class.new(menu)}
+
+  it 'return list of dishes when #see_menu called' do
+    expect(takeaway.see_menu).to eq ({"soup"=>3.22, "garlic bread"=>1.35, "olives"=>2.86})
```

Comment:
Lint/ParenthesesAsGroupedExpression: `(...)` interpreted as grouped expression. (https://github.com/bbatsov/ruby-style-guide#parens-no-spaces)<br>Metrics/LineLength: Line is too long. [90/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)<br>Style/RedundantParentheses: Don't use parentheses around a literal.